src/pages/management/RentalPage.tsx

```
@@ -94,28 +78,36 @@ const modalSubmitFn: Record<string, ModalSubmitFn> = {
 
   useEffect(() => {
     if (isModalOpen === true) return;
     const getTableData = async () => {
-      const res = await getSearchCompanyRental();
+      const res = await getSearchCompanyRental({
+        page: Number(searchParams.get("page")) || 0,
+        size: Number(searchParams.get("size")) || 10,
+      });
       if (res.pass) {
-        setTableData(res.data);
+        setTableData((res.data as any).content);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)